docs/index.md

```diff
@@ -24,6 +24,18 @@ netlify [command] help
 
 Run any Netlify API method
 
+### [blobs](/docs/commands/blobs.md)
```

Comment:
[boulder] -  all of the `docs/commands/blobs.md` links in the CLI docs index go to a broken looking sitemap that doesn't include blobs

<img width="1139" alt="Screenshot 2023-11-15 at 8 34 29 AM" src="https://github.com/netlify/cli/assets/35638702/ac9d6d64-f69a-423e-bd6a-a09b366a7cd7">

Reply:
(woops just posted this in a different comment thread)
https://linear.app/netlify/issue/CT-297/fix-generated-docs-for-cli-recipes-and-blob-command
We've got an issue open for this as it's happening with the `recipes` command as well. I think this is out of scope for this PR, but I'll move this up in our priority list